packages/ember/index.js

```diff
@@ -652,42 +659,6 @@ if (EMBER_GLIMMER_INVOKE_HELPER) {
 }
 Ember._captureRenderTree = captureRenderTree;
 
-if (ENV.EXTEND_PROTOTYPES.String) {
-  String.prototype.htmlSafe = function () {
-    return htmlSafe(this);
-  };
-}
-const deprecateImportFromString = function (
-  name,
-  message = `Importing ${name} from '@ember/string' is deprecated. Please import ${name} from '@ember/template' instead.`
-) {
-  deprecate(message, false, {
```

Comment:
I was thinking you could leave all the code, but change this to `true` with a nice comment...

That would make it really easy to bring it back

Reply:
Totally forgot about that flag 🤦